src/env-inl.h

```diff
@@ -835,7 +836,9 @@ void Environment::set_process_exit_handler(
     PropertyName##_.Set(isolate_, value);                                      \
   }
   PER_ISOLATE_TEMPLATE_PROPERTIES(V)
+  NODE_BUILTIN_ISOLATE_AWARE_MODULES(VM)
```

Comment:
Hmm, it seems a bit weird to call them `NODE_BUILTIN_ISOLATE_AWARE_MODULES()` here, perhaps `NODE_MODULE_PER_ISOLATE_INIT()` would make a bit more sense? Or `NODE_BINDING_PER_ISOLATE_INIT()`